.travis.yml

```diff
@@ -6,20 +6,12 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
Is the flag really needed, or does installation stop abruptly otherwise?

Reply:
It was mentioned in https://github.com/laminas/laminas-xmlrpc/issues/8 that I was to add that, but i'm wondering too

Reply:
I've removed it, tests are all passing without the need for it